app/models/host.rb

```diff
@@ -908,7 +908,7 @@ def reset_discoverable_fields
   def detect_discovered_os(ost)
     # Determine os
     os_type = nil
-    if vmm_vendor == "vmware"
+    if vmm_vendor_display == "vmware" # ??? This will never be true
```

Comment:
@Fryguy yes, I believe it is used for bare-metal and can be `is_vmware?` @gmcculloug please confirm